### ISSUE TITLE
Incorporate #6472 into new release 1.11.2? (Segmentation fault when using KIVY_GL_BACKEND = gl)

### ISSUE THREAD

**Issue by colerd24** [kivy/kivy#6528]
### Versions

* Python: 3.7.3
* OS: Raspbian Buster (4.19.66-v7+) on Raspberry Pi 3B+
* Kivy: 1.11.1
* Kivy installation method: Mostly same as guide, minus gstreamer-plugins 
```bash
sudo apt update
sudo apt upgrade -y
KIVY_PACKAGES="libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev
               pkg-config libgl1-mesa-dev libgles2-mesa-dev
               python3-setuptools libgstreamer1.0-dev git-core
               python3-dev libmtdev-dev
               xclip xsel libjpeg-dev"
               #gstreamer1.0-plugins-{bad,base,good,ugly} Skipped these two
               #gstreamer1.0-{omx,alsa} 
sudo apt install $KIVY_PACKAGES -y

sudo apt install python3-pip -y

python3 -m pip install --upgrade --user pip setuptools
python3 -m pip install --upgrade --user Cython==0.29.10 pillow

python3 -m pip install --upgrade --user kivy
python3 -m pip install --upgrade --user kivy_examples
```

### Description

I am trying to use KIVY_GL_BACKEND set to 'gl' and keep reaching a segmentation fault. I believe this is occurring because of the relative bcm import path in kivy/lib/vidcore_lite/egl.pyx as referenced in #6472. When using 1.11.1 I get the segmentation fault, but I don't have that issue when running the latest master. The program also launches fine when using sdl2. 

I haven't tried merging explicitly #6472 into 1.11.1 as I installed kivy from pip and don't know how to incorporate the fix using a git merge (I was trying to find the file in .local/kivy/lib/vidcore_lite/ but I guess it is compiled somewhere else). If someone could please point me in a direction to test explicitly using that patch, I would be happy to do more testing to confirm that #6472  fixes the issue. 

Would it make sense to create a 1.11.2 release to fix this issue? I know Kivy is used on more platforms than Raspberry Pi so it might be selfish to suggest a whole release dedicated to one platform. If it is not worth the effort, could someone please help point me in the right direction to be able to script this hotfix when installing?

### Code and Logs

I was able to reproduce this issue using the template code located at https://git.io/vM1id. I only modified the imports to add "import os", add "os.environ['KIVY_GL_BACKEND'] = 'gl'", and then set log_level to trace. For what it's worth, I also tried adding os.environ['KIVY_WINDOW'] = 'egl_rpi' with no improvement.

See the following for modifications:
```python
import os

from kivy.config import Config
Config.set('kivy', 'log_level', 'trace')

from kivy.app import App
from kivy.lang import Builder
from kivy.factory import Factory
from kivy.uix.boxlayout import BoxLayout

os.environ['KIVY_GL_BACKEND'] = 'gl'

.... Rest of code from example
```

This is the full log:
```bash
pi@cm-pi:~/kivy_bug $ python3 main.py
[INFO   ] [Logger      ] Record log in /var/log/kivy_logs/kivy_logs_19-09-20_33.txt
[INFO   ] [Kivy        ] v1.11.1
[INFO   ] [Kivy        ] Installed at "/home/pi/.local/lib/python3.7/site-packages/kivy/__init__.py"
[INFO   ] [Python      ] v3.7.3 (default, Apr  3 2019, 05:39:12)
[GCC 8.2.0]
[INFO   ] [Python      ] Interpreter at "/usr/bin/python3"
[INFO   ] [Factory     ] 184 symbols loaded
[DEBUG  ] [Cache       ] register <kv.lang> with limit=None, timeout=None
[TRACE  ] [Lang        ] load file /home/pi/.local/lib/python3.7/site-packages/kivy/data/style.kv
[TRACE  ] [Parser      ] parsing 1295 lines
[TRACE  ] [Parser      ] got directive <kivy 1.0>
[TRACE  ] [Builder     ] build rule for <Label>
[TRACE  ] [Builder     ] build rule for <-Button,-ToggleButton>
[TRACE  ] [Builder     ] build rule for <BubbleContent>
[TRACE  ] [Builder     ] build rule for <BubbleButton>
[TRACE  ] [Builder     ] build rule for <Slider>
[TRACE  ] [Builder     ] build rule for <ProgressBar>
[TRACE  ] [Builder     ] build rule for <SplitterStrip>
[TRACE  ] [Builder     ] build rule for <Scatter>
[TRACE  ] [Builder     ] build rule for <RelativeLayout>
[TRACE  ] [Builder     ] build rule for <Image,AsyncImage>
[TRACE  ] [Builder     ] build rule for <EffectWidget>
[TRACE  ] [Builder     ] build rule for <TabbedPanelContent>
[TRACE  ] [Builder     ] build rule for <TabbedPanelStrip>
[TRACE  ] [Builder     ] build rule for <StripLayout>
[TRACE  ] [Builder     ] build rule for <TabbedPanelHeader>
[TRACE  ] [Builder     ] build rule for <Selector>
[TRACE  ] [Builder     ] build rule for <TextInput>
[TRACE  ] [Builder     ] build rule for <TextInputCutCopyPaste>
[TRACE  ] [Builder     ] build rule for <CodeInput>
[TRACE  ] [Builder     ] build rule for <TreeViewNode>
[TRACE  ] [Builder     ] build rule for <TreeViewLabel>
[TRACE  ] [Builder     ] build rule for <StencilView>
[TRACE  ] [Builder     ] build rule for <FileChooserListLayout>
[TRACE  ] [Builder     ] build rule for <FileChooserListView>
[TRACE  ] [Builder     ] build template for [FileListEntry@FloatLayout+TreeViewNode]
[TRACE  ] [Builder     ] build rule for <FileChooserIconLayout>
[TRACE  ] [Builder     ] build rule for <FileChooserIconView>
[TRACE  ] [Builder     ] build template for [FileIconEntry@Widget]
[TRACE  ] [Builder     ] build rule for <FileChooserProgress>
[TRACE  ] [Builder     ] build rule for <Switch>
[TRACE  ] [Builder     ] build rule for <ModalView>
[TRACE  ] [Builder     ] build rule for <Popup>
[TRACE  ] [Builder     ] build rule for <SpinnerOption>
[TRACE  ] [Builder     ] build rule for <Spinner>
[TRACE  ] [Builder     ] build rule for <ActionBar>
[TRACE  ] [Builder     ] build rule for <ActionView>
[TRACE  ] [Builder     ] build rule for <ActionSeparator>
[TRACE  ] [Builder     ] build rule for <ActionButton,ActionToggleButton>
[TRACE  ] [Builder     ] build rule for <ActionLabel>
[TRACE  ] [Builder     ] build rule for <ActionGroup>
[TRACE  ] [Builder     ] build rule for <ActionCheck>
[TRACE  ] [Builder     ] build rule for <ActionPreviousImage@Image>
[TRACE  ] [Builder     ] build rule for <ActionPreviousButton@Button>
[TRACE  ] [Builder     ] build rule for <ActionPrevious>
[TRACE  ] [Builder     ] build rule for <ActionGroup>
[TRACE  ] [Builder     ] build rule for <ActionOverflow>
[TRACE  ] [Builder     ] build rule for <ActionDropDown>
[TRACE  ] [Builder     ] build template for [AccordionItemTitle@Label]
[TRACE  ] [Builder     ] build rule for <AccordionItem>
[TRACE  ] [Builder     ] build rule for <SettingSpacer>
[TRACE  ] [Builder     ] build rule for <SettingItem>
[TRACE  ] [Builder     ] build rule for <SettingBoolean>
[TRACE  ] [Builder     ] build rule for <SettingString>
[TRACE  ] [Builder     ] build rule for <SettingPath>
[TRACE  ] [Builder     ] build rule for <SettingOptions>
[TRACE  ] [Builder     ] build rule for <SettingTitle>
[TRACE  ] [Builder     ] build rule for <SettingSidebarLabel>
[TRACE  ] [Builder     ] build rule for <SettingsPanel>
[TRACE  ] [Builder     ] build rule for <Settings>
[TRACE  ] [Builder     ] build rule for <InterfaceWithSidebar>
[TRACE  ] [Builder     ] build rule for <InterfaceWithSpinner>
[TRACE  ] [Builder     ] build rule for <MenuSpinner>
[TRACE  ] [Builder     ] build rule for <MenuSidebar>
[TRACE  ] [Builder     ] build rule for <ContentPanel>
[TRACE  ] [Builder     ] build rule for <InterfaceWithTabbedPanel>
[TRACE  ] [Builder     ] build rule for <ScrollView>
[TRACE  ] [Builder     ] build rule for <VideoPlayerPreview>
[TRACE  ] [Builder     ] build rule for <VideoPlayerAnnotation>
[TRACE  ] [Builder     ] build rule for <VideoPlayer>
[TRACE  ] [Builder     ] build rule for <CheckBox>
[TRACE  ] [Builder     ] build rule for <ScreenManager>
[TRACE  ] [Builder     ] build rule for <ColorPicker_Input@TextInput>
[TRACE  ] [Builder     ] build rule for <ColorPicker_Label@Label>
[TRACE  ] [Builder     ] build rule for <ColorPicker_Selector@BoxLayout>
[TRACE  ] [Builder     ] build rule for <ColorWheel>
[TRACE  ] [Builder     ] build rule for <ColorPicker>
[DEBUG  ] [Cache       ] register <kv.image> with limit=None, timeout=60
[DEBUG  ] [Cache       ] register <kv.atlas> with limit=None, timeout=None
[TRACE  ] [Image       ] Unable to use <gif> as loader!
[TRACE  ]
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/kivy/core/__init__.py", line 135, in core_register_libs
    level=0)
  File "/home/pi/.local/lib/python3.7/site-packages/kivy/core/image/img_ffpyplayer.py", line 7, in <module>
    import ffpyplayer
ModuleNotFoundError: No module named 'ffpyplayer'
[INFO   ] [Image       ] Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
[DEBUG  ] [Cache       ] register <kv.texture> with limit=1000, timeout=60
[DEBUG  ] [Cache       ] register <kv.shader> with limit=1000, timeout=3600
[TRACE  ] [Parser      ] parsing 21 lines
[TRACE  ] [Parser      ] got directive <import rand_float random.uniform>
[TRACE  ] [Builder     ] build rule for <CustomLabel@Label>
[TRACE  ] [Builder     ] build rule for <Test>
[DEBUG  ] [App         ] Loading kv <./test.kv>
[DEBUG  ] [App         ] kv <./test.kv> not found
[TRACE  ] [Parser      ] parsing 19 lines
[TRACE  ] [Builder     ] build rule for <-CoverBehavior>
[DEBUG  ] [Window      ] Ignored <egl_rpi> (import error)
[TRACE  ]
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/kivy/core/__init__.py", line 63, in core_select_lib
    fromlist=[modulename], level=0)
  File "/home/pi/.local/lib/python3.7/site-packages/kivy/core/window/window_egl_rpi.py", line 12, in <module>
    from kivy.lib.vidcore_lite import bcm, egl
ImportError: cannot import name 'bcm' from 'kivy.lib.vidcore_lite' (/home/pi/.local/lib/python3.7/site-packages/kivy/lib/vidcore_lite/__init__.py)
[INFO   ] [Window      ] Provider: sdl2(['window_egl_rpi'] ignored)
[INFO   ] [GL          ] Using the "OpenGL" graphics system
[DEBUG  ] [GL          ] glShaderBinary is not available
[INFO   ] [GL          ] Backend used <gl>
Segmentation fault

```

**Comment by matham** [comment]
We won't be making a 1.11.2 release, unless a core-dev volunteers to do that, which is exceedingly unlikely. Since this is a support request and a bug report, you should ask for help on how to merge the fixes on discord because we don't use issues for support requests.

**Comment by colerd24** [comment]
As an update: I haven't merged [#6472](https://github.com/kivy/kivy/pull/6472) into 1.11.1 and built it from scratch and the issue was resolved. Just wanted to update the thread in case any other Raspberry Pi users encounter this bug.
